Fix bold markdown formatting.

tyclintw opened the pull request:
Fix bold markdown formatting as well as adjust sentence to be parallel with the HDInsight Hive Activity documentation.

azurecla commented:
Hi __@diegoviso__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tyclintw committed:
Fix bold markdown formatting.

Fix bold markdown formatting as well as adjust sentence to be parallel with the HDInsight Hive Activity documentation.
